app/views/jobs/index.html.erb

```diff
@@ -19,9 +20,11 @@
           </tr>
 
         <% @jobs.each do |job| %>
+          <% agent = Agent.find(YAML.load(job.handler).args[0]) %>
```

Comment:
Maybe use `find_by_id`, so it does not raise an exception when displaying a job of a deleted agent?